modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -59,6 +59,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 == Additions
 {productname} 6.5 also includes the following additions:
 
+=== Toolbar and menu items were not being disabled correctly when selecting noneditable elements inside a `contenteditable= "false"` root.
+//#TINY-9669
+
+In previous versions of {productname}, an issue was identified where toolbar and menu items were not being disabled correctly when selecting `contenteditable= "false"` contents inside a `contenteditable= "false"` root.
+
+As a consequence, toolbar buttons and menu items would appear to be enabled even though the buttons and menu items were in a disabled state based on the users current selection.
+
+A fix was applied in {productname} 6.5, that now disables the toolbar buttons and menu items based on the users selection. In addition, if `contenteditable= "true"` contents are selected, the toolbars and menu items are re-enabled.
```

Comment:
```suggestion
=== Toolbar and menu items were not disabled correctly when selecting elements with a `contenteditable="false"` attribute inside an editor root with a `contenteditable="false"` attribute set.
//#TINY-9669

In previous versions of {productname}, an issue was identified where toolbar and menu items were not presenting as disabled when the contents of elements with a `contenteditable= "false"` attribute were selected from inside a editor root with a `contenteditable="false"` attribute set.

In this circumstance, toolbar buttons and menu items appeared as enabled even though the buttons and menu items were in a disabled state based on the user’s current selection.

{productname} 6.5 now explicitly disables toolbar buttons and menu items based on the user’s selection.

In addition, if the contents of elements with a `contenteditable="true"` attribute are selected in the same circumstance, the toolbars and menu items are re-enabled.
```

1. Edits and copy-edits.
2. The technically accurate phrases edited into place: _`contenteditable="false"` attribute_ and _`contenteditable="true"` attribute_